src/patternfly/components/MenuToggle/menu-toggle.scss

```
@@ -337,6 +341,33 @@
   &.pf-m-danger {
     --#{$menu-toggle}--BorderColor: var(--#{$menu-toggle}--m-danger--BorderColor);
     --#{$menu-toggle}__status-icon--Color: var(--#{$menu-toggle}--m-danger__status-icon--Color);
+
+    @media screen and (prefers-reduced-motion: no-preference) {
+      transform: translate3d(0, 0, 0);
+      animation-name: menu-toggle-m-danger-motion;
+      animation-duration: var(--#{$menu-toggle}--m-danger--AnimationDuration--Transform);
+      animation-timing-function: var(--#{$menu-toggle}--m-danger--AnimationTimingFunction--Transform);
+      animation-fill-mode: both;
+      
+      .#{$menu-toggle}__status-icon {
+        color: var(--#{$menu-toggle}__status-icon--Color, inherit);
+        animation-name: menu-toggle-status-icon-fade-in;
```

to avoid conflicts with different versions of patternfly loaded together, let's use the component var with the version in it

```suggestion
        animation-name: #{$menu-toggle}-status-icon-fade-in;
```